### DISCUSSION TITLE
Midi output not working for both ALSA and JACK.

### DISCUSSION THREAD

**Discussion by king112ola** [hydrogen-music/hydrogen#1635] [Hydrogen on Linux]
Hello Hydrogen team :wave: 

Receiving Midi Works fine. However, I cannot send midi to other applications, e.g. Ardour.

Version: 
Hydrogen 1.1.1
Ardour 6.9.0
Fedora 36

**Comment by trebmuh** [comment]
Closing here in favor of an issue. See #1636 .
